app/routes/news._index/route.mdx

```diff
@@ -27,6 +27,28 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2025
 
+    <CollectionItem
+      className="maxw-none grid-container"
+      variantComponent={
+        <CollectionCalendarDate datetime={'April 14, 2025'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars Archive Viewable by Event </Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        The Circulars archive now provides two options for viewing and searching:
+          - Circulars View
+            - Standard view of Circulars sorted inversely by Circular ID.
+            - Search bar searches for text strings (e.g. 250101A) or Circulars numbers (e.g. 12345).
+          - Events View
+            - Lists astronomical events (e.g. GRB250101A, S250101, EP250101a) with toggle to show all Circulars associated with that event and a link to a page that lists all Circulars in sequence similar to the GCN Classic Circulars Archive.
```

Comment:
It's not a toggle. You can expand and collapse events to view the Circulars that belong to them.